common/cli-utils.js

```diff
@@ -1,96 +1,119 @@
-'use strict';
+"use strict";
 
 require("ts-node/register"); /* enable node during migration */
-require('dotenv').config();
+require("dotenv").config();
 
-const fs = require('fs');
-const { Option } = require('commander');
+const fs = require("fs");
+const { Option } = require("commander");
 
 // A path to the chain configuration files
 const CHAIN_CONFIG_PATH = `${__dirname}/../axelar-chains-config/info`;
 
 // A list of available chain environments which are the names of the files in the CHAIN_CONFIG_PATH
-const CHAIN_ENVIRONMENTS = fs.readdirSync(CHAIN_CONFIG_PATH).map((chainName) => chainName.split('.')[0]);
+const CHAIN_ENVIRONMENTS = fs
+  .readdirSync(CHAIN_CONFIG_PATH)
+  .map((chainName) => chainName.split(".")[0]);
 
 const addEnvOption = (program, defaultValue) => {
-    program.addOption(
-        new Option('-e, --env <env>', 'environment')
-            .choices(CHAIN_ENVIRONMENTS)
-            .default(defaultValue || 'testnet')
-            .makeOptionMandatory(true)
-            .env('ENV'),
-    );
+  program.addOption(
```

Comment:
can we change indentation later? it's going to create a conflict for all the open PRs. we should merge some PRs first and then migrate if we want